src/login/login.component.css

```diff
@@ -0,0 +1,52 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.center {
+  text-align: center;
+}
+
+.logo {
+  margin-bottom: 3rem;
+}
+
+.submit-btn {
```

Comment:
FYI: to avoid having to `styles["submit-btn"]` instead of the shorter `styles.submitBtn`, the css modules community usually camel cases its css classes instead of using hyphens. This is explicitly [said in the css modules spec](https://github.com/css-modules/css-modules#naming)

Both work and what you have here isn't wrong. Sharing just for knowledge.

Reply:
Thanks